app/angular/service/conversorService.js

```diff
@@ -139,11 +139,11 @@ const logicConversorService = (ConceptualService, $uibModal) => {
 					template: '<conversion-option-modal title="$ctrl.title" options="$ctrl.options" summary="$ctrl.summary" close="$close(result)"></conversion-option-modal>',
 					controller: function () {
 						const $ctrl = this;
-						$ctrl.title = `Atenção - Assistente de conversão - Especialização`;
-						$ctrl.summary = `Não foi possível realizar essa conversão pois a tabela ${getExtensionRootName(extension)} possui outras conexões. Escolha outra opção`;
+						$ctrl.title = $filter('translate')("Attention - Conversion Assistant - Specialization");
+						$ctrl.summary = $filter('translate')("It was not possible to perform this conversion because table X has other connections. Choose another option.", { table: getExtensionRootName(extension) });
```

Comment:
```suggestion
						$ctrl.summary = $filter('translate')("It was not possible to perform this conversion because table TABLE_NAME has other connections. Choose another option.", { table: getExtensionRootName(extension) });
```